packages/react-core/src/components/Drawer/DrawerPanelContent.tsx

```diff
@@ -76,6 +79,34 @@ export const DrawerPanelContent: React.FunctionComponent<DrawerPanelContentProps
     }
   }, [isStatic, isExpanded]);
 
+  const calcValueNow = () => {
+    let splitterPos;
+    let drawerSize;
+
+    if (isInline) {
+      splitterPos =
+        splitterRef.current.getBoundingClientRect().right - drawerContentRef.current.getBoundingClientRect().left;
+      drawerSize = drawerRef.current.getBoundingClientRect().right - drawerRef.current.getBoundingClientRect().left;
+    } else if (position === 'right') {
+      splitterPos =
+        splitterRef.current.getBoundingClientRect().left - drawerContentRef.current.getBoundingClientRect().left;
+      drawerSize =
+        drawerContentRef.current.getBoundingClientRect().right - drawerContentRef.current.getBoundingClientRect().left;
+    } else if (position === 'left') {
+      splitterPos =
+        splitterRef.current.getBoundingClientRect().right - drawerContentRef.current.getBoundingClientRect().left;
+      drawerSize =
+        drawerContentRef.current.getBoundingClientRect().right - drawerContentRef.current.getBoundingClientRect().left;
+    } else if (position === 'bottom') {
+      splitterPos =
+        drawerContentRef.current.getBoundingClientRect().bottom - splitterRef.current.getBoundingClientRect().top;
+      drawerSize =
+        drawerContentRef.current.getBoundingClientRect().bottom - drawerContentRef.current.getBoundingClientRect().top;
+    }
+
+    return Math.round(splitterPos / drawerSize) * 100;
```

Comment:
This is missing a couple of parentheses just before the `Math.round`; right now it's always returning 100 or 0, nothing ever in between:

```suggestion
    return Math.round((splitterPos / drawerSize) * 100);
```

Reply:
Nice catch!!

Reply:
I had it like that and the linter removed the brackets.

Reply:
That's odd that it'd remove it. Could we try disabling eslint for that line in particular?